src/common/components/hosts/HostsTable.tsx

```diff
@@ -13,24 +13,28 @@ import { HostDetail } from '../../../common/components/hosts/HostRowDetail';
 import { WithTestID } from '../../types';
 import EmptyState from '../ui/uiState/EmptyState';
 import { usePagination } from './usePagination';
+import { getHostStr } from '../clusters/utils';
 
 const getHostId = (host: Host) => host.id;
 
 type HostsTableEmptyStateProps = {
   setDiscoveryHintModalOpen?: HostsNotShowingLinkProps['setDiscoveryHintModalOpen'];
+  isSNO?: boolean;
 };
 
 export const HostsTableEmptyState: React.FC<HostsTableEmptyStateProps> = ({
   setDiscoveryHintModalOpen,
+  isSNO = false,
 }) => (
   <EmptyState
     icon={ConnectedIcon}
-    title="Waiting for hosts..."
+    title={`Waiting for ${getHostStr(isSNO)}...`}
```

Comment:
Since we already have humanize-plus as a dependency, I am inclined to being explicit and treat the `hosts` string just as another string that needs pluralization and use:
```suggestion
    title={`Waiting for ${pluralize(+isSNO, 'host')}...`}
```

Reply:
@jkilzi also mentioned this library, I will use it instead